app/javascript/react/components/MonsterTile.js

```diff
@@ -0,0 +1,78 @@
+  import React, {useState, useEffect} from "react"
+  import { Redirect } from "react-router-dom"
+
+const MonsterTile = props => {
+  const { monster } = props
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+  const [votes, setVotes] = useState(monster.votes)
+  const changeRedirect = event => {
+    setShouldRedirect(true)
+  }
+  const updateVotes = async (direction) =>{
```

Comment:
I love the enthusiasm! Let's come back to this voting feature after some core functionality has been built out